### ISSUE TITLE
JS unit test execution is broken

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/activity#137]
```
PhantomJS 2.1.1 (Linux 0.0.0) ActivityCollection query url does not filter by default FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 1 of 8 (1 FAILED) (0 secs / 0.004 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityCollection query url does not filter by default - page2 FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 2 of 8 (2 FAILED) (0 secs / 0.004 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityCollection query url filters by id and type when specified FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 3 of 8 (3 FAILED) (0 secs / 0.005 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityCollection query url filters by id and type when specified - page2 FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 4 of 8 (4 FAILED) (0 secs / 0.006 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityTabView rendering reloads matching activities when setting file info model FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	ReferenceError: Can't find variable: sinon in tests/js/activitytabviewSpec.js (line 18)

	tests/js/activitytabviewSpec.js:18:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'tabView.setFileInfo') in tests/js/activitytabviewSpec.js (line 32)

	tests/js/activitytabviewSpec.js:32:11

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'fetchStub.restore') in tests/js/activitytabviewSpec.js (line 27)

	tests/js/activitytabviewSpec.js:27:13

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 5 of 8 (5 FAILED) (0 secs / 0.008 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityTabView rendering renders loading icon while fetching activities FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	ReferenceError: Can't find variable: sinon in tests/js/activitytabviewSpec.js (line 18)

	tests/js/activitytabviewSpec.js:18:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'tabView.setFileInfo') in tests/js/activitytabviewSpec.js (line 40)

	tests/js/activitytabviewSpec.js:40:11

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'fetchStub.restore') in tests/js/activitytabviewSpec.js (line 27)

	tests/js/activitytabviewSpec.js:27:13

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 6 of 8 (6 FAILED) (0 secs / 0.009 secs)

PhantomJS 2.1.1 (Linux 0.0.0) ActivityTabView rendering renders activities FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	ReferenceError: Can't find variable: sinon in tests/js/activitytabviewSpec.js (line 18)

	tests/js/activitytabviewSpec.js:18:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'tabView.setFileInfo') in tests/js/activitytabviewSpec.js (line 89)

	tests/js/activitytabviewSpec.js:89:11

	loaded@http://localhost:9876/context.js:162:17

	TypeError: undefined is not an object (evaluating 'fetchStub.restore') in tests/js/activitytabviewSpec.js (line 27)

	tests/js/activitytabviewSpec.js:27:13

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 7 of 8 (7 FAILED) (0 secs / 0.016 secs)

PhantomJS 2.1.1 (Linux 0.0.0) Filter tests something FAILED

	ReferenceError: Can't find variable: sinon in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 150)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:150:21

	loaded@http://localhost:9876/context.js:162:17

	TypeError: null is not an object (evaluating 'fakeServer.restore') in /home/travis/build/nextcloud/server/core/js/tests/specHelper.js (line 174)

	/home/travis/build/nextcloud/server/core/js/tests/specHelper.js:174:13

	loaded@http://localhost:9876/context.js:162:17

PhantomJS 2.1.1 (Linux 0.0.0): Executed 8 of 8 (8 FAILED) (0 secs / 0.016 secs)

JS 2.1.1 (Linux 0.0.0): Executed 8 of 8 (8 FAILED) ERROR (0.03 secs / 0.016 secs)
```

@ChristophWurst can you maybe help?

**Comment by ChristophWurst** [comment]
> Can't find variable: sinon

Is the sinon lib included?

**Comment by nickvergessen** [comment]
Yeah no it was not, it used a shipped version in core. But that was replaced.
Fixed with #138
